Cannot find name 'OperationObject'

**Description**

When attempting to use the `openapi-fetch` library, it imports `openapi-typescript-helpers`, which tries to use the `OperationObject` type declaration which is not in scope. It seems to be missing an import of this type.

**Reproduction**

```
$ npx tsc
node_modules/openapi-typescript-helpers/index.d.ts:19:51 - error TS2304: Cannot find name 'OperationObject'.

19 export type PathItemObject = { [M in HttpMethod]: OperationObject } & { parameters?: any };
                                                     ~~~~~~~~~~~~~~~


Found 1 error in node_modules/openapi-typescript-helpers/index.d.ts:19

$ cat src/index.ts 
import createClient from "openapi-fetch";
import { paths } from "./api"; // (generated from openapi-typescript)

async function main() {
    const { GET } = createClient<paths>({ baseUrl: "http://127.0.0.1:8000" });
    const { data, error } = await GET("/user", {});
    console.log(data);
}

main();

$ cat tsconfig.json 
{
  "compilerOptions": {
    "target": "es6",
    "module": "commonjs",
    "outDir": "dist",
    "lib": [
        "es2017",
        "dom"
    ]
  }
}
```

Versions

- node: v20.5.1
- typescript: 5.2.2
- openapi-fetch: 0.7.4
- openapi-typescript-helpers: 0.0.1
- openapi-typescript: 6.5.3

**Expected result**

No compilation errors

**Checklist**

- [X] I’m willing to open a PR (see [CONTRIBUTING.md](https://github.com/drwpow/openapi-fetch/blob/main/CONTRIBUTING.md))

--- a/.changeset/early-feet-report.md
+++ b/.changeset/early-feet-report.md
@@ -1,0 +1,5 @@
+---
+"openapi-typescript-helpers": patch
+---
+
+Fix type bug

--- a/packages/openapi-typescript-helpers/index.d.ts
+++ b/packages/openapi-typescript-helpers/index.d.ts
@@ -15,6 +15,13 @@ export type ErrorStatus = 500 | '5XX' | 400 | 401 | 402 | 403 | 404 | 405 | 406 
 export type PathsWithMethod<Paths extends Record<string, PathItemObject>, PathnameMethod extends HttpMethod> = {
   [Pathname in keyof Paths]: Paths[Pathname] extends { [K in PathnameMethod]: any } ? Pathname : never;
 }[keyof Paths];
+/** DO NOT USE! Only used only for OperationObject type inference */
+export interface OperationObject {
+  parameters: any;
+  params?: { query?: Record<string, unknown> };
+  requestBody: any; // note: "any" will get overridden in inference
+  responses: any;
+}
 /** Internal helper used in PathsWithMethod */
 export type PathItemObject = { [M in HttpMethod]: OperationObject } & { parameters?: any };
 /** Return `responses` for an Operation Object */

--- a/packages/openapi-typescript-helpers/tsconfig.json
+++ b/packages/openapi-typescript-helpers/tsconfig.json
@@ -1,7 +1,8 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "rootDir": "."
+    "rootDir": ".",
+    "skipLibCheck": false
   },
   "include": ["."]
 }

